calculator.css

```diff
@@ -0,0 +1,40 @@
+body {background-color:#6497B1; font-family: Arial, Helvetica, sans-serif; font-size: 16px; color: white;}
+
+b {color: white; font-size: 16px;}
+o {color: black; font-size: 50px; text-decoration: none;}
+a:visited {color:white; text-decoration: none;}
+            h1 {font-family: Arial, Helvetica, sans-serif; color: #38505c; font-size: 50px; letter-spacing: -3px; padding-bottom:;}
+            h2 {font-family: Arial, Helvetica, sans-serif; color: palegoldenrod; font-style: italic; font-size: 25px;}
+            p {width: 500px; text-align: center;}
+            #sqr { width: 150px; height: 150px; background-color: 
+          lightgreen;}
+            #crcl { width: 150px; height: 150px; background-color: moccasin; border-radius: 90px;}
```

Comment:
CSS formatting can be improved a lot. Please see some online examples if needed.

https://css-tricks.com/different-ways-to-format-css/